api/EXT_shader_texture_lod.json

```diff
@@ -5,40 +5,41 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/EXT_shader_texture_lod",
         "support": {
           "chrome": {
-            "version_added": true
+            "version_added": "38"
           },
           "chrome_android": {
-            "version_added": true
+            "version_added": "38"
           },
           "edge": {
-            "version_added": "17"
+            "version_added": "17",
+            "version_removed": "79"
```

Comment:
No Edge support?

Reply:
Unfortunately no, this is one of the extensions Edge doesn't support.